main.js

```diff
@@ -0,0 +1,20 @@
+document.querySelector('body li').remove();
+
+document.querySelector('span').remove();
+
+let articles = document.querySelectorAll('article');
+
+for (let i = 0; i < articles.length; i++) {
+    if (articles[i].textContent.includes('Stop Planning')) {
+        articles[i].remove();
+    }
+}
```

Comment:
Interesting way to do this one, I like it! 

here's another way to do it in case you're curious:
`articles[articles.length - 1].remove()` <-- removing the last element of the node list.

Reply:
Thanks Sam! Also, thanks for the other way to do it, I appreciate it. Definitely a cleaner and shorter way, I like it better than mine!